src/app/controllers/checkout/checkout.controller.ts

```diff
@@ -0,0 +1,210 @@
+module webShoes {
+  'use script';
+  class Product {
+    id         : number;
+    img1       : string;
+    img2       : string;
+    img3       : string;
+    img4       : string;
+    manufactor : string;
```

Comment:
@lucasBgabriel you're right, but now i think it would be better to keep `manufactor` since all classes are expecting this key haha